tileset.go

```diff
@@ -51,7 +51,7 @@ func readTileset(r io.Reader) (*Tileset, error) {
 		log.WithError(err).Error("readTileset: could not decode to Tileset")
 		return nil, err
 	}
-	return &t, nil
+	return validate(&t)
```

Comment:
Validate will never need to modify the tileset, so no need for pointer here
```suggestion
	return validate(t)
```